Only import zmesh if called

j6k4m8 opened the pull request:
You don't need zmesh installed unless you're meshing something.

movestill commented on intern/service/mesh/service.py:
```diff
@@ -13,21 +13,23 @@
-    millimeters = 1000000, 
-    centimeters = 10000000,
+
+    nm = (1,)
```
Why are these tuples, now?

j6k4m8 replied:
oh shoot i must have accidentally linted this full file. let me fix that...

j6k4m8 replied:
although — it looks like this comma-notation in the enum means that the interpreter was reading these types as `Tuple[int]`s anyway. The linter just added parentheses for clarity. I _think_ that's not what we wanted, but check this out:

```python
from enum import IntEnum

class X(IntEnum):
    a = (1,)
    b = (2,)

X.a == 1 # True
```

`IntEnum` does some weird stuff here (details on SO [here](https://stackoverflow.com/a/26125425/979255)); i have a feeling we've been using tuples all along...

j6k4m8 commented:
Updated the docstring, fixed tuple→int, and moved zmesh import to inside the invocation so you don't need it to install intern. Ready for your thoughts :)
